sql/2020/05_Media/big_non_custom_metrics.sql

```diff
@@ -0,0 +1,26 @@
+#standardSQL
+# non custom metrics sql that uses regexp on response bodies
+# TODO: replace sample database with the real one when done
+# img src vs data-uri
+# count rel=preconnect
+SELECT client,
+  count(0) as total,
+  countif(has_img_data_uri) as has_img_data_uri,
+  countif(has_img_src) as has_img_src,
+  countif(rel_preconnect) as rel_preconnect
+FROM
+  (
+  SELECT
+    client,
+    page,
+    regexp_contains(body, r'<img[^><]*src=(?:\"|\')*data[:]image/(?:\"|\')*[^><]*>') as has_img_data_uri,
+    regexp_contains(body, r'<img[^><]*src=[^><]*>') as has_img_src,
+    regexp_contains(body, r'<link[^><]*rel=(?:\"|\')*preconnect/(?:\"|\')*[^><]*>') as rel_preconnect
+  FROM
+    `httparchive.sample_data.summary_response_bodies_firstHtml`
```

Comment:
I did not run this yet, only tested on sample data. Anyway if we decide to get last year sqls and run them now we will end up spending a lot too.

Reply:
By the way, Rick replied here https://github.com/HTTPArchive/almanac.httparchive.org/pull/1327#issuecomment-724097296 that it might not cost $127 per query as I suspected, but much less.

Reply:
Ah that's true. Still if we've got them cheaper let's use them. Haven't had a chance to dig into the custom metrics yet to confirm but hopefully will later today.

Reply:
OK looks like `<img src>` data is not available in a custom metric so I was wrong about that.

Resolving this. Though think we should look at SUM as well as COUNTIFs as per other feedback and need to change to the real table before merging.